composer.json

```diff
@@ -14,7 +14,9 @@
         }
     },
     "require": {
-        "php": ">=5.3.0"
+        "php": ">=5.3.0",
+        "alcaeus/mongo-php-adapter": "^1.0",
+        "paragonie/random_compat": "^2.0"
```

Comment:
I think this should be `>=2` instead, per <https://github.com/paragonie/random_compat#version-99999> so that the vast majority of users (which is PHP 7.0, 7.1, 7.2 or later) don't end up installing this is.

Alternatively, we may want to do a major release of xhgui sometime soon that drops PHP 5.x support to focus on PHP 7.1+ instead, at which point this wouldn't be needed.

As of today, PHP 5.6 and PHP 7.0 have reached end-of-life, per [https://php.net/supported-versions](https://secure.php.net/supported-versions.php).

Reply:
Dropping PHP5 support is something that should happen in the near-future. It might require a bunch of work to update the dependencies that xhgui has though.

Reply:
I think version bump is out of scope of this PR, and `>=2` should be used for now.